### PR TITLE
[Serverless] bump integration test dependencies

### PR DESCRIPTION
**Pull request by maxday**
### What does this PR do?

Bump fasthttp

### Motivation

Ever greening

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: cmd/serverless/main.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func main() {
 		os.Exit(-1)
 	}
 
-	// handle SIGTERM
+	// handle SIGTERM signal
```
Comment (maxday) [inline cmd/serverless/main.go:76]: small change to trigger integration tests

Comment (IvanTopolcic) [reply cmd/serverless/main.go:76]: Not sure if it applies here but usually when I need to commit something to trigger the CI I run a 'git commit --allow-empty' to push an empty commit.

**Review by IvanTopolcic — APPROVED** [review]
LGTM

File: cmd/serverless/main.go (hunk 1 above)
Comment (maxday) [reply cmd/serverless/main.go:76]: Yes, but this does not apply there. 
We trigger integration on serverless only if specific files are touch ( see : https://github.com/DataDog/datadog-agent/blob/main/.github/workflows/serverless-integration.yml#L5 )
I think this is about the change and we will run integration tests on all commits in a near future

**Pull request merged**